Sources/Core/CompilerProtocols.swift

```diff
@@ -38,3 +38,17 @@ public protocol RawRepresentable {
   var rawValue: RawValue { get }
   init?(rawValue: RawValue)
 }
+
+public protocol _ExpressibleByBuiltinStringLiteral {
+  init(
```

Comment:
Please don't break immediately after the `(`, prefer to bin-pack the arguments.

Reply:
What column limit would you like to set? Would something like 100 or 120 work?